tests/select.spec.ts

```diff
@@ -1,20 +1,204 @@
 import { test, expect } from '@playwright/test';
+import { waitForCustomEvent } from './lib/helper.ts';
 
-test('user is able to make a selection with mouse', async ({ page }) => {
-  await page.goto('/test.html');
+test.beforeEach(async ({page}) => {
+  await page.goto('/select-test.html')
+})
 
-  await expect(page.getByLabel(/bobbins/i)).toBeVisible()
+test('Given an options attribute, on select the options should be visible', async ({ page }) => {
```

Comment:
can we split this test up? feels like 3 separate tests to me: options initially not visible, options visible on focus/click, click on option works properly

Reply:
The reason i didn't is because separating out tests would just duplicate boilerplate setup code and because this test essentially tests the default expected behavior of the component it made sense to just put it together. In general, I am anti separating tests when its duplicating setup stuff because in the L-R it slows down CI but can separate it out if there is a strong opinion to do so.